x/valset-pref/keeper_test.go

```diff
@@ -27,15 +27,15 @@ func (suite *KeeperTestSuite) PrepareDelegateToValidatorSet() []types.ValidatorP
 		},
 		{
 			ValOperAddress: valAddrs[1],
-			Weight:         sdk.NewDecWithPrec(332, 3), // 0.332
+			Weight:         sdk.NewDecWithPrec(332, 3), // 0.33
 		},
 		{
 			ValOperAddress: valAddrs[2],
 			Weight:         sdk.NewDecWithPrec(12, 2), // 0.12
 		},
 		{
 			ValOperAddress: valAddrs[3],
-			Weight:         sdk.NewDecWithPrec(348, 3), // 0.348
+			Weight:         sdk.NewDecWithPrec(348, 3), // 0.35
```

Comment:
I didn't realize why this changed until the end. Maybe add some comment here explaining that we round after second digit in val basic

Reply:
will do